app/static/js/trackVolunteers.js

```diff
@@ -74,6 +75,7 @@ $(document).ready(function() {
       url: `/removeVolunteerFromEvent/${username}/${eventId}`,
       type: "POST",
       success: function(s) {
+         $(".removeVolunteer").prop("disabled", true)
```

Comment:
This disable is unnecessary, as the page is about to reload (and you already disabled it on click anyway). What you need is to re-enable the button if you don't reload the page

Reply:
resolved